File: examples/minimal.pl
#!/usr/bin/env perl

use Glib::Object::Introspection;
Glib::Object::Introspection->setup(
    basename => 'OpenDCS',
    version => '0.1',
    package => 'OpenDCS');

$object = OpenDCS::Object->new();
$object->set_hash(0);
print $object->get_hash();
exit 0;

File: examples/minimal.py
#!/usr/bin/python3

from gi.repository import GLib
from gi.repository import OpenDCS

import sys

class DcsExample(object):
    def __init__(self):
        self.dcsobject = OpenDCS.Object.new()
        self.dcsobject.set_hash(0)
        print >>sys.stderr, self.dcsobject.get_hash()

if __name__ == "__main__":
    DcsExample()
